src/main/kotlin/lotto/domain/Lottery.kt

```diff
@@ -0,0 +1,32 @@
+package lotto.domain
+
+import lotto.model.LottoErrorCode
+
+@JvmInline
+value class Lottery(private val numbers: Set<LottoNumber>) : Set<LottoNumber> by numbers {
+
+    init {
+        require(value = size == ALLOW_LOTTO_NUMBER_COUNT) {
```

Comment:
`value =`은 어떤 의도로 들어간건지 알 수 있을까요? 6개인지만 체크하면 되지 않나 해서요.

```suggestion
        require(size == ALLOW_LOTTO_NUMBER_COUNT) {
```

Reply:
namedParameter가 컨벤션이다보니, 습관적으로 달아주고 있습니다!
kotlin, require의 파라미터 네임입니다.